CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ option(ENABLE_COVERAGE "Enable coverage reports (includes enabling all tests and
 option(ENABLE_WERROR "Treat all build warnings as errors" OFF)
 option(ENABLE_COLOR_TEMP "Include colour temperature specific code in the build" ON)
 
+if (ENABLE_COLOR_TEMP)
+    option (ENABLE_RDA "Enable RDA (remote desktop awareness)" ON)
+endif ()
```

Comment:
Options should not depend on other build option being true. Rather extend this option's description: "Enable RDA (remote desktop awareness), only taking effect if built with ENABLE_COLOR_TEMP=ON.

Reply:
Done.